DigitalLearningSolutions.Data.Tests/Services/CourseServiceTests.cs

```diff
@@ -199,10 +208,124 @@ public void VerifyAdminUserCanAccessCourse_should_return_false_with_incorrect_ca
         {
             // Given
             A.CallTo(() => courseDataService.GetCourseValidationDetails(A<int>._))
-                .Returns((1, 1));
+                .Returns((1, 1, false));
+
+            // When
+            var result = courseService.VerifyAdminUserCanManageCourse(1, 1, 2);
+
+            // Then
+            A.CallTo(() => courseDataService.GetCourseValidationDetails(1))
+                .MustHaveHappenedOnceExactly();
+            result.Should().BeFalse();
+        }
+
+        [Test]
+        public void VerifyAdminUserCanAccessCourse_should_return_null_when_course_does_not_exist()
+        {
+            // Given
+            A.CallTo(() => courseDataService.GetCourseValidationDetails(A<int>._))
+                .Returns((null, null, null));
+
+            // When
+            var result = courseService.VerifyAdminUserCanManageCourse(1, 1, 2);

```

Comment:
Test name `VerifyAdminUserCanAccessCourse_should_return_null_when_course_does_not_exist`
should change to match the new target of VerifyAdminUserCanManageCourse:
`VerifyAdminUserCanManageCourse_should_return_null_when_course_does_not_exist`